pillar/apps/ocw-production.sls

```diff
@@ -1,7 +1,19 @@
+
+{% set dbhost_ipaddr_data =
+     salt.saltutil.runner('mine.get',
+                          tgt='G@roles:ocw-db and G@environment:ocw-production',
+                          fun='network.ip_addrs',
+                          tgt_type='compound')
+%}
+{% set dbhost_ipaddr = dbhost_ipaddr_data[1][0] %}
```

Comment:
The return value of the `mine.get` is a dictionary of the form:
```
{
    "ocw-production-cms-db1": [
        "10.100.0.12"
    ], 
    "ocw-production-cms-db2": [
        "10.100.0.7"
    ]
}
```
So the first index should be a string to retrieve the proper instance.

Reply:
OK! Does commit 88f8b53 look any better?